service/src/main/java/io/camunda/service/UserServices.java

```diff
@@ -52,4 +52,6 @@ public CompletableFuture<UserRecord> createUser(
             .setEmail(email)
             .setPassword(password));
   }
+
+  public record CreateUserRequest(String username, String name, String email, String password) {}
```

Comment:
❗Although we still don't have the update user in service/controller but I'm wondering if we need the same dto for it, or does it look different. (We were using the PUT method before and in that case they were same but I'm not sure about dto when using PATCH)

Reply:
We might reuse it. But we can rename it once we implement that. No need to consider it in this PR 🙂